### DISCUSSION TITLE
Capturing input values from inspector

### DISCUSSION THREAD

**Discussion by kvinita** [clientIO/joint#2580] [Q&A]
### Introduction

I have a cell that has nested properties like 
```
ownerInfo: {
    ownerName: string;
  };
```
The ownerName is bound to a text input field in inspector. I need to capture the data that is typed into the input field. 
I came across 'change' event that is triggered when user changes property through inspector. 

```
this.graph.on('change:ownerInfo/ownerName', (element: joint.dia.Element) => {
      console.log(element);
    });
```
Is that the correct way to capture the value typed into ownerName(which is a nested field) ? FYI, I'm only facing issue while capturing nested fields. Non-nested fields work.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
Please see https://github.com/clientIO/joint/discussions/1754#discussioncomment-3273307.
